pandas/tests/resample/test_datetime_index.py

```diff
@@ -1687,7 +1687,7 @@ def f(data, add_arg):
 
     # Testing dataframe
     df = DataFrame({"A": 1, "B": 2}, index=date_range("2017", periods=10))
-    result = df.groupby("A").resample("D").agg(f, multiplier)
+    result = df.groupby("A").resample("D").agg(f, multiplier).astype(float)
     expected = df.groupby("A").resample("D").mean().multiply(multiplier)
     # TODO: GH 41137
```

Comment:
@rhshadrach does this PR take care of the TODO comment here?

Reply:
Yes - thanks, I've opened #44374